### PR TITLE
Extend fromQuery function

### PR DESCRIPTION
**Pull request by NelinD**
Added the ability to form `select` `option` with `data-*` attributes.
Added the ability to form `option` text through the passed function.
Use
```
		Former::select('foo')->fromQuery(
			$collection,
			function($model) {
				//return select option item text
			},
			[
				'value' => 'id',
				'data-test' => function($model) {
					//return select option item data-* attribute value
				}
			]
		);
```
The backward compatibility with existing code.

### CODE REVIEW

**Comment by greabock** [comment]
great!

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 # Dependencies
 composer.lock
 vendor
+/composer.phar
+/phpunit.phar
```
Comment (tortuetorche) [inline .gitignore]: @NelinD Good job :clap:
This pull request looks good, can you remove this 2 lines, please?
Then I'll merged it.

**Comment by NelinD** [comment]
Done.

File: .gitignore (hunk 1 above)
Comment (NelinD) [inline .gitignore]: Done.

**Comment by NelinD** [comment]
Done.

File: .gitignore (hunk 1 above)
Comment (NelinD) [inline .gitignore]: Done.

**Comment by tortuetorche** [comment]
Many thanks for this contribution @NelinD

**Comment by tortuetorche** [comment]
Now we need to write some docs about this new feature.

**Comment by NelinD** [comment]
https://github.com/formers/former/wiki/Anatomy
Done.

**Comment by tortuetorche** [comment]
:+1:

**Pull request merged**